go.sum

```diff
@@ -4,19 +4,15 @@ github.com/Masterminds/semver v1.4.2 h1:WBLTQ37jOCzSLtXNdoo8bNM8876KhNqOKvrlGITg
 github.com/Masterminds/semver v1.4.2/go.mod h1:MB6lktGJrhw8PrUyiEoblNEGEQ+RzHPF078ddwwvV3Y=
```

Comment:
Try and keep go.mod/sum changes out of PRs unless the PR explicitly requires it.

Reply:
Wait, you told me before to include it in PR's -_-. I remember asking you specifically about it.

Reply:
No, go.mod/go.sum specify what dependencies the codebase requires, the changes you've made are not affected by any dependencies so these files shouldn't be changed during your patch.

Your editor/tooling might attempt to be helpful by auto updating dependencies by running `go mod tidy` in the background but this is not really useful. If you notice go.mod/sum files while running `git add` then reset their state back so your branch is using the same dependencies as upstream.